library/Fission/Web/DNS.hs

```diff
@@ -4,19 +4,20 @@ module Fission.Web.DNS
   ) where
 
 import           Database.Esqueleto
-import           Network.IPFS.CID.Types
 import           Servant
 
 import           Fission.Authorization
 import           Fission.Models
 import           Fission.Prelude
 
 import           Fission.URL                 as URL
-import           Fission.User.Username.Types
 import           Fission.Web.Error           as Web.Err
 
+import           Fission.User.Username.Types
 import qualified Fission.User.Modifier       as User
 
+import           Network.IPFS.CID.Types
```

Comment:
No need to change, but FYI the import layout is `general -> specific use case`. This import would be up at line 8.

Reply:
ah cool :+1: hadn't quite figured out the rhyme or reason to import layout